tests/intel_fpga_test.sh

```diff
@@ -44,7 +44,7 @@ run_sample() {
     echo -e ${3} | $PYTHON_BINARY ${1}.py ${@:4}
 
     if [ $? -ne 0 ]; then
-        bail "$1 (${RED}Wrong emulation result${NC})"
```

Comment:
Why remove?

Reply:
This is printed no matter why the executable fails, 90% of the time it's _not_ due to wrong emulation results ;-) We could only reintroduce something like this if we had some convention for return codes